transformer_engine/pytorch/ops/fuser.py

```diff
@@ -352,7 +352,9 @@ def __call__(
             basic_op_kwargs = [{} for _ in range(len(self._basic_ops))]
 
         # Flatten list of parameters
-        params = [param for param in op.parameters() for op in self._basic_ops]
+        params = []
+        for op in self._basic_ops:
+            params.extend(op.parameters())
```

Comment:
It turns out the right syntax for nested list comprehensions is to put the outer iterator first:
```suggestion
        params = [param for op in self._basic_ops for param in op.parameters()]
```